packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -25,7 +26,8 @@ const int _kDefaultSize = 1000;
 /// Generally this class is not used directly. The [ImageProvider] class and its
 /// subclasses automatically handle the caching of images.
 class ImageCache {
-  final Map<Object, ImageStreamCompleter> _cache = <Object, ImageStreamCompleter>{};
+  final Map<Object, ImageStreamCompleter> _pendingImages = <Object, ImageStreamCompleter>{};
```

Comment:
You don't seem to actually use this map.

Reply:
Fixed - should be used for de-duping multiple requests for the same image key